profiles/offtkp.json

```diff
@@ -0,0 +1,6 @@
+{
+ "image": "https://raw.githubusercontent.com/OFFTKP/OFFTKP/master/profile.png",
+ "issueId": 66,
+ "name": "Paris Oplopoios",
+ "username": "offtkp"
```

Comment:
It looks like the indentation is wrong here.
```suggestion
  "image": "https://raw.githubusercontent.com/OFFTKP/OFFTKP/master/profile.png",
  "issueId": 66,
  "name": "Paris Oplopoios",
  "username": "offtkp"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient